projects/observability/src/shared/dashboard/widgets/waterfall/waterfall-widget-renderer.component.ts

```diff
@@ -67,12 +70,18 @@ import { MarkerSelection, WaterfallChartComponent } from './waterfall/waterfall-
           [activeTabLabel]="this.activeTabLabel"
           (closed)="this.closeSheet()"
         >
-          <ht-summary-value
-            data-sensitive-pii
-            icon="${IconType.SpanId}"
-            label="Span ID"
-            [value]="this.selectedData!.id"
-          ></ht-summary-value>
+          <ht-explore-filter-link
+            class="filter-link"
+            [paramsOrUrl]="getExploreNavigationParams | htMemoize: this.selectedData | async"
+            htTooltip="See traces in Explorer"
```

Comment:
it looks like this is going to see spans, not traces

Reply:
@jyothishjose6190 Please update this.